benchmarks/bm_runner.py

```diff
@@ -112,6 +111,12 @@ def _prep_data_gen_env() -> None:
             ]
         )
 
+        echo("Installing iris-test-data into data generation environment ...")
```

Comment:
Could we skip this if there is an existing setting ?
i.e. `if "OVERRIDE_TEST_DATA_REPOSITORY" not in environ`

Reply:
47fe9a8a66089c7a6d8193f0763aca027cc04060

Reply:
As discussed offline ...
From one angle, it looks like the fix will re-install every time as it can't set OVERRIDE.. permanently in the caller's env.
But actually it is OK because the download is in a fixed location, and [once it exists there it will not be re-downloaded](https://github.com/SciTools/iris/blob/47fe9a8a66089c7a6d8193f0763aca027cc04060/benchmarks/bm_runner.py#L98-L99).